### ISSUE TITLE
Prime Symbol Not Being Read Correctly in JAWS (when MathCAT is not enabled)

### ISSUE THREAD

**Issue by jenmarsala** [daisy/math-a11y#41]
<h1>Description</h1>
When creating math content using Word's Equation Editor, there are issues with JAWS ignoring the prime symbol (apostrophe) when MathCAT is NOT enabled. The prime symbol is important in Calculus as it is used to identify derivatives.

<h2>Expected Behavior</h2>
The apostrophe should be read as "prime" when within math content.

<h2>Actual Behavior</h2>
Five methods of typing "f prime" are listed below, some of which read correctly with JAWS and others which do not. The issues are with methods 3 and 4.<br/><br/>Under JAWS Options, Early Adopter Program - make sure that MathCAT is NOT enabled.</br>

<ol><br/>

<li><strong>Method 1:</strong> Type f then apostrophe then spacebar.<br/><br/>JAWS correctly reads this as "math content f prime."</li><br/>

<li><strong>Method 2:</strong> Type f then apostrophe, then press enter.<br/><br/>JAWS correctly reads this as "math content f prime."</li><br/>

<li><mark><strong>Method 3 (problematic):</strong></mark> Type f then apostrophe, then the right arrow (to exit the equation editor). People may use this method when they are typing additional math content on the same line.<br/><br/>When JAWS punctuation settings are set to None, Some, or Most, JAWS ignores the apostrophe and reads, "math content f" -- missing the apostrophe altogether.<br/><br/>When JAWS punctuation settings are set to All, JAWS reads "math content f apostrophe." This is better but not ideal, as JAWS should ideally read "math content f prime."</li><br/>

<li><mark><strong>Method 4 (problematic):</strong></mark> Type f then \prime then spacebar.<br/><br/>The results are the same as for Method 3:<br/><br/>When JAWS punctuation settings are set to None, Some, or Most, JAWS ignores the apostrophe and reads, "math content f" -- missing the apostrophe altogether.<br/><br/> When JAWS punctuation settings are set to All, JAWS reads "math content f apostrophe." This is better but not ideal (as it should ideally read "math content f prime."</li><br/>

<li><strong>Method 5 (works - but is a nonobvious workaround):</strong> Type f then \prime and then press spacebar TWICE. <br/><br/>JAWS correctly reads this as "math content f prime" -- but a user would be very unlikely to press spacebar twice, especially since the prime symbol displays visually as soon as the spacebar is pressed one time.<br/></li>
</ol>

<h1>Tested Version Information</h1>

<h2>Windows OS</h2>
OS Version:<br/>
-- OS Name: Microsoft Windows 10 Enterprise<br/>
-- Version: 10.0.19045 Build 19045 <br/>
-- Locale: United States<br/><br/>
Microsoft Word version:<br/>
-- Microsoft 365 Apps for Enterprise<br/>
-- Version 2410 (Build 18129.20158 Click-to-Run)<br/><br/>
JAWS Version:<br/>
-- JAWS 2025.2410.106<br/>

**Comment by brichwin** [comment]
I could not reproduce this using the following environment:
- Windows 11 Version 24H2 (Build 26100.6584)
- Microsoft® Word for Microsoft 365 MSO (Version 2510 Build 16.0.19309.20000) 64-bit
- JAWS 2025.2508.120 fresh install (default settings)

@jenmarsala Would you please check to see if this is still an issue on your end?

**Comment by jenmarsala** [comment]
It's possible that this issue may still exist for older versions of JAWS (2024, 2023, etc.) since MathCAT was enabled by default in the JAWS 2025 version. If anyone has an older version of JAWS to test out this issue, please do so. I used to keep the older versions on my machine, but it seems that I have removed them all.
